db-scripts/01_create_views.sql

```diff
@@ -0,0 +1,34 @@
+/******************************************************\

```

Comment:
@tylerjamesf - Like a temporary query to begin stitching it all together and see what like / don't like.